rest-api-guidelines/guides/api-testing-ci-environment.md

```diff
@@ -31,63 +24,28 @@ The following must be available in the CI environment before testing:
     dredd v2.2.5 (Darwin 16.4.0; x64)
```

Comment:
current version of Dredd is @13.0.0. I guess we should update that ;D

Reply:
Updated.